proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -363,15 +363,18 @@ message ConsumerAddrsToPrune {
   AddressList consumer_addrs = 3;
 }
 
-// ConsumerValidator is used to facilitate epoch-based transitions. It contains relevant info for
-// a validator that is expected to validate on a consumer chain during an epoch.
-message ConsumerValidator {
+// ConsensusValidator is used to express a validator that
+// should be validating on a chain. 
+// It contains relevant info for
+// a validator that is expected to validate on 
+// either the provider or a consumer chain.
+message ConsensusValidator {
```

Comment:
The latest `ConsumerValidator` contains a [`JoinHeight`](https://github.com/cosmos/interchain-security/blob/ac73a52aec1132ccff2a8dd5b7d88315a0951c53/proto/interchain_security/ccv/provider/v1/provider.proto#L370) that is not really applicable to a provider validator address. Besides the `JoinHeight` I do not see us extended `ConsumerValidator` further in the foreseeable future, but wondering whether having two types of validators `ProviderValidator` and `ConsumerValidator` would make sense as well (?)

Reply:
Hm, ok, I can see that.
I'll investigate whether it makes sense, I want to be able to reuse as much code as possible, but yeah, maybe they get too distinct with that change

Reply:
I decided to keep it as ConsensusValidator to reduce the amount of duplicated code. since it seems we will not add extra consumer specific fields in the near future, I think it should be ok, and we can refactor later if it becomes tedious